File: src/node/Consumer.js
var kafka = require('kafka-node'),
Consumer = kafka.Consumer,
client = new kafka.KafkaClient('kafka:9094'),
consumer = new Consumer(
    client,
    [
        { topic: 'test', partition: 0 }
    ],
    {
        autoCommit: false
    }
);

consumer.on('message', function(message) {
  console.log(message);
})

File: src/consumer.py
from kafka import KafkaConsumer

topic = "test"

# Subscribe to a regex topic pattern
consumer = KafkaConsumer(
    topic,
    bootstrap_servers=['kafka:9092']
    #client_id='stylers-kafka-01'
)

for message in consumer:
    print("%s:%d:%d: key=%s value=%s" % (message.topic, message.partition,
                                         message.offset, message.key,
                                         message.value))
